### DISCUSSION TITLE
Pl-6 and md:grid-cols does not work on v3.0.8

### DISCUSSION THREAD

**Discussion by devLeopar** [tailwindlabs/tailwindcss#6812] [Help]
**What version of Tailwind CSS are you using?**

v3.0.8

**What build tool (or framework if it abstracts the build tool) are you using?**

Nextjs 10.1.3

**What version of Node.js are you using?**

v16.13.1

**What browser are you using?**

Chrome

**What operating system are you using?**

Linux Mint 20.2 Cinnamon

It is reproduced on localhost right now

**Describe your issue**

After upgrading my tailwindcss from 2.1.2 to 3.0.8 and applied all upgrade steps correctly, I realized that some of classnames are not working!
Up-to now, I can figure it out that "pl-6" and "md:grid-cols-{number}" can not be applied to the style of corresponding element. I had to replace them with "pl-4" and "lg:grid-cols-{number}"
 
What can be the reason behind that?

**Comment by adamwathan** [comment]
Have you properly configured your `content` to include the files that contain those classes?

https://tailwindcss.com/docs/content-configuration

**Reply by devLeopar** [reply]
tailwind.config.js
```
module.exports = {
  content: [
    "./src/pages/**/*.{js,ts,jsx,tsx}",
    "./src/components/**/*.{js,ts,jsx,tsx}",
    "./src/modules/**/*.{js,ts,jsx,tsx}"
  ],
  theme: {
    boxShadow: {
      sm: "0 1px 6px -1px rgba(0, 0, 0, 0.04), 0 2px 4px -1px rgba(0, 0, 0, 0.04);",
      lg: "0 5px 20px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)",
      md: "0 1px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06)",
      xl: "0 10px 25px -5px rgba(0, 0, 0, 0.1), 0 10px 10px -5px rgba(0, 0, 0, 0.04)",
      DEFAULT:
        "0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06)",
      box: "0px 16px 48px rgba(0, 142, 240, 0.16);",
      boxComplete: "0px 16px 48px rgba(26, 211, 113, 0.16);",
      active:
        "0 1px 6px -1px rgba(0, 142, 240, 0.2), 0 1px 6px -1px rgba(0, 142, 240, 0.2)"
    },
    maxWidth: {
      10: "10rem",
      18: "18rem"
    },
    container: {  // I've commented out that plugin but it did not help
      screens: {
        sm: "100%",
        md: "100%",
        lg: "1128px",
        xl: "1128px"
      }
    },
    extend: {
      gridTemplateRows: {
        2: "repeat(2, minmax(0, 1fr))",
        layout: "6rem 1fr"
      },
      gridTemplateColumns: {
        "auto-fit": "repeat(auto-fit, minmax(200px, 1fr))",
        auto: "repeat(auto-fit, minmax(6rem, 1fr));"
      }
    }
  },
```
tsconfig.json
```
{
  "compilerOptions": {
    "baseUrl": ".",
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "paths": {
      "*": ["src/*"]
    },
  },
  "include": [
    "next-env.d.ts",
    "**/*.ts",
    "**/*.tsx"
  ],
  "exclude": [
    "node_modules"
  ]
}

```

Here are the setup that I've been running

**Reply by adamwathan** [reply]
And what is the full path to the file containing “pl-6”?

**Comment by devLeopar** [comment]
src/modules/layouts/ContentGrid.tsx

On that file, `pl-4` is working `lg:grid-cols-4` is working but `pl-6` and `md:grid-cols-4` not. I don't know but it does not seems it is related to path of file
